### PR TITLE
response: record not found addition

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi leonardo.r@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
please rebase

File: examples/app.py [hunk 1]
```diff
@@ -77,6 +77,8 @@
 InvenioRecords(app)
 InvenioOAIServer(app)
 
+app.register_blueprint(blueprint)
```
Comment (jirikuncar) [inline examples/app.py:86]: not needed

File: examples/app.py [hunk 2]
```diff
@@ -63,13 +63,13 @@
...
 from invenio_oaiserver import InvenioOAIServer
 from invenio_oaiserver.minters import oaiid_minter
+from invenio_oaiserver.views.server import blueprint
 
 # Create Flask application
 app = Flask(__name__)
 app.config.update(
     SQLALCHEMY_DATABASE_URI=os.getenv('SQLALCHEMY_DATABASE_URI',
                                       'sqlite:///app.db'),
-    SERVER_NAME="app",
```
Comment (jirikuncar) [inline examples/app.py]: already fixed

File: invenio_oaiserver/response.py [hunk 3]
```diff
@@ -226,17 +227,24 @@ def header(parent, identifier, datestamp, sets=None, deleted=False):
...
+
+        header(
+            e_getrecord,
+            identifier=str(pid.object_uuid),
+            datestamp=record.updated,
+        )
+        e_metadata = SubElement(e_getrecord,
+                                etree.QName(NS_OAIPMH, 'metadata'))
+        e_metadata.append(record_dumper(record.json))
+    except PIDDoesNotExistError:
```
Comment (jirikuncar) [inline invenio_oaiserver/response.py]: do not handle exception here but in view

File: examples/app.py [hunk 4]
```diff
@@ -82,6 +83,8 @@
 # InvenioSearch(app)
 InvenioOAIServer(app)
 
+app.register_blueprint(blueprint)
```
Comment (jirikuncar) [inline examples/app.py:86]: not needed

File: invenio_oaiserver/views/server.py [hunk 5]
```diff
@@ -77,7 +79,13 @@ def extract_errors():
 @use_args(make_request_validator)
 def response(args):
     """Response."""
-    e_tree = getattr(xml, args['verb'].lower())(**args)
+    try:
+        e_tree = getattr(xml, args['verb'].lower())(**args)
+    except PIDDoesNotExistError:
```
Comment (jirikuncar) [inline invenio_oaiserver/views/server.py:84]: do not handle exception inside the function

Comment (hachreak) [inline invenio_oaiserver/views/server.py:84]: where do I handle the error? Will I make a decorator?

**Pull request merged**